SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -349,6 +368,22 @@ void USpatialReceiver::OnAuthorityChange(const Worker_AuthorityChangeOp& Op)
 		return;
 	}
 
+	if (Op.component_id == SpatialConstants::ENTITY_ACL_COMPONENT_ID
+		&& Op.authority == WORKER_AUTHORITY_AUTHORITATIVE)
+	{
+		if (StaticComponentView->GetComponentData<SpatialGDK::AuthorityIntent>(Op.entity_id) == nullptr)
+		{
+			// We're seeing an authority gain for the ACL component, but we can only process those if we can see the auth intent component.
+			// Buffer and handle once we see the add auth intent.
+			PendingAclAuthorityChanges.Add(Op.entity_id, Op);
```

Comment:
If this op and the metadata component add op come in the same update, is there any validation that this op is processed first?

Reply:
Oh I see, this is checked against the component data itself.